### ISSUE TITLE
Cannot deploy to Heroku. `No such file or directory: '/tmp/build_c6d21c3b/translate/dist'`

### ISSUE THREAD

**Issue by surapunoyousei** [mozilla/pontoon#2978]
```
kit-3.3.2 typing-extensions-4.5.0 tzdata-2022.7 tzlocal-4.2 uhashring-2.1 unidecode-1.3.2 urllib3-1.26.7 vine-5.0.0 wcwidth-0.2.5 webencodings-0.5.1 werkzeug-2.0.2 whitenoise-5.2.0 wsgi-sslify-1.0.1
remote: -----> $ python manage.py collectstatic --noinput
remote:        Traceback (most recent call last):
remote:          File "/tmp/build_c6d21c3b/manage.py", line 16, in <module>
remote:            execute_from_command_line(sys.argv)
remote:          File "/app/.heroku/python/lib/python3.9/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
remote:            utility.execute()
remote:          File "/app/.heroku/python/lib/python3.9/site-packages/django/core/management/__init__.py", line 413, in execute
remote:            self.fetch_command(subcommand).run_from_argv(self.argv)
remote:          File "/app/.heroku/python/lib/python3.9/site-packages/django/core/management/base.py", line 354, in run_from_argv
remote:            self.execute(*args, **cmd_options)
remote:          File "/app/.heroku/python/lib/python3.9/site-packages/django/core/management/base.py", line 398, in execute
remote:            output = self.handle(*args, **options)
remote:          File "/app/.heroku/python/lib/python3.9/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 187, in handle
remote:            collected = self.collect()
remote:          File "/app/.heroku/python/lib/python3.9/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 105, in collect
remote:            for path, storage in finder.list(self.ignore_patterns):
remote:          File "/app/.heroku/python/lib/python3.9/site-packages/django/contrib/staticfiles/finders.py", line 130, in list
remote:            for path in utils.get_files(storage, ignore_patterns):
remote:          File "/app/.heroku/python/lib/python3.9/site-packages/django/contrib/staticfiles/utils.py", line 23, in get_files
remote:            directories, files = storage.listdir(location)
remote:          File "/app/.heroku/python/lib/python3.9/site-packages/django/core/files/storage.py", line 330, in listdir
remote:            for entry in os.scandir(path):
remote:        FileNotFoundError: [Errno 2] No such file or directory: '/tmp/build_c6d21c3b/translate/dist'
remote: 
remote:  !     Error while running '$ python manage.py collectstatic --noinput'.
remote:        See traceback above for details.
remote: 
remote:        You may need to update application code to resolve this error.
remote:        Or, you can disable collectstatic for this application:
remote: 
remote:           $ heroku config:set DISABLE_COLLECTSTATIC=1
```

I followed the Deployment documentation and pushed to Heroku to build, but an error occurred and I could not deploy.

We tried to resolve the issue, but could not, so we are opening an issue.

Also, before this error, I got an error that mysqlclient was not installed, but I add `mysqlclient==2.2.0 \ ~` to `requirements.txt` file and that resolved the issue.

**Comment by mathjazz** [comment]
@surapunoyousei Did you try to deploy with the purple Deploy to Heroku button?

**Comment by surapunoyousei** [comment]
> @surapunoyousei Did you try to deploy with the purple Deploy to Heroku button?

I use heroku command via Heroku CLI.

This error output from this.

But, I did try to use the purple Deploy to Heroku button because, before error. However, Button is not working.
```
 postdeploy exit code was not 0

+ export SITE_URL=http://localhost:8000

+ SITE_URL=http://localhost:8000

++ echo mc3.dev.ec2.memcachier.com:11211

++ cut -f1 -d:

+ MEMCACHE_SERVER=mc3.dev.ec2.memcachier.com

++ echo mc3.dev.ec2.memcachier.com:11211

++ cut -f2 -d:

+ MEMCACHE_PORT=11211

+ nc -z -w5 mc3.dev.ec2.memcachier.com 11211

+ CELERY_ALWAYS_EAGER=True

+ ./manage.py sync_projects --no-commit

Scheduling sync for project Pontoon Intro.

Syncing project pontoon-intro.

Pulling source changes for project pontoon-intro started.

Git: [Errno 2] No such file or directory: '/app/media/projects/pontoon-intro/mozilla/pontoon-intro.git'

Pulling source changes for project pontoon-intro complete.

Retrieving changed files for: /app/media/projects/pontoon-intro/mozilla/pontoon-intro.git/locales/en-US:None

Changed files in Pontoon Intro repository and Pontoon, relevant for enabled locales: {'messages.properties': []}

Relative paths in Pontoon Intro that need to be synced: dict_keys(['messages.properties'])

Sending new string notifications for project Pontoon Intro.

New string notifications for project Pontoon Intro sent.

Synced sources for project pontoon-intro.

Syncing translations for project: pontoon-intro

Retrieving changed files for: /app/media/projects/pontoon-intro/mozilla/pontoon-intro.git:None

Changed files in Pontoon Intro repository, all: ['CODE_OF_CONDUCT.md', 'README.md', 'LICENSE', 'locales/zh-HK/messages.properties', 'locales/trs/messages.properties', 'locales/es-ES/.keep', 'locales/es-ES/messages.properties', 'locales/ar/.keep', 'locales/ar/messages.properties', 'locales/scn/messages.properties', 'locales/is/.keep', 'locales/is/messages.properties', 'locales/ff/.keep', 'locales/ff/messages.properties', 'locales/br/.keep', 'locales/br/messages.properties', 'locales/frp/messages.properties', 'locales/ru/.keep', 'locales/ru/messages.properties', 'locales/ca/.keep', 'locales/ca/messages.properties', 'locales/zh-TW/.keep', 'locales/zh-TW/messages.properties', 'locales/ms/.keep', 'locales/ms/messages.properties', 'locales/nl/.keep', 'locales/nl/messages.properties', 'locales/sl/.keep', 'locales/sl/messages.properties', 'locales/ro/.keep', 'locales/ro/messages.properties', 'locales/be/.keep', 'locales/be/messages.properties', 'locales/ilo/.keep', 'locales/ilo/messages.properties', 'locales/de/.keep', 'locales/de/messages.properties', 'locales/eo/.keep', 'locales/eo/messages.properties', 'locales/csb/.keep', 'locales/csb/messages.properties', 'locales/nb-NO/.keep', 'locales/nb-NO/messages.properties', 'locales/pt-BR/.keep', 'locales/pt-BR/messages.properties', 'locales/ig/.keep', 'locales/ha/.keep', 'locales/ga-IE/.keep', 'locales/ga-IE/messages.properties', 'locales/bn/messages.properties', 'locales/ku/.keep', 'locales/ku/messages.properties', 'locales/mix/messages.properties', 'locales/cy/.keep', 'locales/cy/messages.properties', 'locales/mai/.keep', 'locales/mai/messages.properties', 'locales/es/.keep', 'locales/es/messages.properties', 'locales/ast/.keep', 'locales/ast/messages.properties', 'locales/ga/.keep', 'locales/ga/messages.properties', 'locales/fr/.keep', 'locales/fr/messages.properties', 'locales/es-CL/.keep', 'locales/es-CL/messages.properties', 'locales/kpv/messages.properties', 'locales/ltg/.keep', 'locales/ltg/messages.properties', 'locales/kk/.keep', 'locales/kk/messages.properties', 'locales/oc/.keep', 'locales/oc/messages.properties', 'locales/uk/.keep', 'locales/uk/messages.properties', 'locales/ka/.keep', 'locales/ka/messages.properties', 'locales/tsz/.keep', 'locales/tsz/messages.properties', 'locales/el/.keep', 'locales/el/messages.properties', 'locales/es-AR/.keep', 'locales/es-AR/messages.properties', 'locales/ne-NP/.keep', 'locales/ne-NP/messages.properties', 'locales/fa/.keep', 'locales/fa/messages.properties', 'locales/ta/.keep', 'locales/ta/messages.properties', 'locales/mn/.keep', 'locales/mn/messages.properties', 'locales/as/.keep', 'locales/as/messages.properties', 'locales/en-GB/.keep', 'locales/en-GB/messages.properties', 'locales/ckb/messages.properties', 'locales/mg/.keep', 'locales/cs/.keep', 'locales/cs/messages.properties', 'locales/lg/.keep', 'locales/lg/messages.properties', 'locales/sq/.keep', 'locales/sq/messages.properties', 'locales/tl/.keep', 'locales/tl/messages.properties', 'locales/kn/.keep', 'locales/kn/messages.properties', 'locales/lo/.keep', 'locales/lo/messages.properties', 'locales/th/.keep', 'locales/th/messages.properties', 'locales/zh-CN/.keep', 'locales/zh-CN/messages.properties', 'locales/hr/.keep', 'locales/hr/messages.properties', 'locales/et/.keep', 'locales/et/messages.properties', 'locales/km/.keep', 'locales/km/messages.properties', 'locales/ach/.keep', 'locales/ach/messages.properties', 'locales/vi/.keep', 'locales/vi/messages.properties', 'locales/zu/.keep', 'locales/zu/messages.properties', 'locales/af/.keep', 'locales/af/messages.properties', 'locales/bn-BD/.keep', 'locales/bn-BD/messages.properties', 'locales/id/.keep', 'locales/id/messages.properties', 'locales/ml/.keep', 'locales/ml/messages.properties', 'locales/ee/.keep', 'locales/su/messages.properties', 'locales/son/.keep', 'locales/son/messages.properties', 'locales/nso/.keep', 'locales/nso/messages.properties', 'locales/gl/.keep', 'locales/gl/messages.properties', 'locales/lt/.keep', 'locales/lt/messages.properties', 'locales/am/messages.properties', 'locales/gn/.keep', 'locales/gn/messages.properties', 'locales/nn-NO/.keep', 'locales/nn-NO/messages.properties', 'locales/hsb/.keep', 'locales/hsb/messages.properties', 'locales/ace/messages.properties', 'locales/mr/.keep', 'locales/mr/messages.properties', 'locales/sr/.keep', 'locales/sr/messages.properties', 'locales/pl/.keep', 'locales/pl/messages.properties', 'locales/rm/.keep', 'locales/rm/messages.properties', 'locales/bm/.keep', 'locales/kab/messages.properties', 'locales/wo/.keep', 'locales/wo/messages.properties', 'locales/it/.keep', 'locales/it/messages.properties', 'locales/az/.keep', 'locales/az/messages.properties', 'locales/lb/messages.properties', 'locales/pa/.keep', 'locales/ln/.keep', 'locales/en-US/messages.properties', 'locales/or/.keep', 'locales/or/messages.properties', 'locales/lij/.keep', 'locales/lij/messages.properties', 'locales/zgh/messages.properties', 'locales/gu-IN/.keep', 'locales/gu-IN/messages.properties', 'locales/tr/.keep', 'locales/tr/messages.properties', 'locales/ht/.keep', 'locales/ht/messages.properties', 'locales/anp/messages.properties', 'locales/xh/.keep', 'locales/xh/messages.properties', 'locales/pt-PT/.keep', 'locales/pt-PT/messages.properties', 'locales/da/.keep', 'locales/da/messages.properties', 'locales/dsb/.keep', 'locales/dsb/messages.properties', 'locales/sk/.keep', 'locales/sk/messages.properties', 'locales/hu/.keep', 'locales/hu/messages.properties', 'locales/pt/.keep', 'locales/pt/messages.properties', 'locales/sw/.keep', 'locales/sw/messages.properties', 'locales/ta-LK/.keep', 'locales/ta-LK/messages.properties', 'locales/fy/.keep', 'locales/bn-IN/.keep', 'locales/bn-IN/messages.properties', 'locales/te/.keep', 'locales/te/messages.properties', 'locales/fy-NL/.keep', 'locales/fy-NL/messages.properties', 'locales/eu/.keep', 'locales/eu/messages.properties', 'locales/si/.keep', 'locales/si/messages.properties', 'locales/ko/.keep', 'locales/ko/messages.properties', 'locales/sv-SE/.keep', 'locales/sv-SE/messages.properties', 'locales/vec/messages.properties', 'locales/jv/messages.properties', 'locales/ia/messages.properties', 'locales/pai/messages.properties', 'locales/pa-IN/.keep', 'locales/pa-IN/messages.properties', 'locales/ak/.keep', 'locales/ak/messages.properties', 'locales/fi/.keep', 'locales/fi/messages.properties', 'locales/fur/.keep', 'locales/fur/messages.properties', 'locales/bg/.keep', 'locales/bg/messages.properties', 'locales/tg/messages.properties', 'locales/hi-IN/.keep', 'locales/hi-IN/messages.properties', 'locales/es-MX/.keep', 'locales/es-MX/messages.properties', 'locales/my/.keep', 'locales/my/messages.properties', 'locales/bs/.keep', 'locales/bs/messages.properties', 'locales/an/.keep', 'locales/an/messages.properties', 'locales/en-ZA/.keep', 'locales/en-ZA/messages.properties', 'locales/mk/.keep', 'locales/mk/messages.properties', 'locales/ppl/messages.properties', 'locales/ur/.keep', 'locales/ur/messages.properties', 'locales/uz/.keep', 'locales/uz/messages.properties', 'locales/cak/.keep', 'locales/cak/messages.properties', 'locales/ja/.keep', 'locales/ja/messages.properties', 'locales/hus/messages.properties', 'locales/gd/.keep', 'locales/gd/messages.properties', 'locales/hy-AM/.keep', 'locales/hy-AM/messages.properties', 'locales/kw/messages.properties', 'locales/lv/.keep', 'locales/lv/messages.properties', 'locales/he/.keep', 'locales/he/messages.properties', 'locales/yo/.keep']

Changed files in Pontoon Intro repository, relevant for enabled locales: {'messages.properties': [<Locale: English>]}

Changed files in Pontoon Intro repository and Pontoon, relevant for enabled locales: {'messages.properties': [<Locale: English>]}

Relative paths in Pontoon Intro that need to be synced: dict_keys(['messages.properties'])

Synced locale en-GB for project pontoon-intro.

Synced translations for project pontoon-intro in locales en-GB.

Sync complete.

[0mTraceback (most recent call last):

  File "/app/.heroku/python/lib/python3.9/site-packages/django/db/models/options.py", line 608, in get_field

    return self.fields_map[field_name]

KeyError: 'url'



During handling of the above exception, another exception occurred:



Traceback (most recent call last):

  File "/app/./manage.py", line 16, in <module>

    execute_from_command_line(sys.argv)

  File "/app/.heroku/python/lib/python3.9/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line

    utility.execute()

  File "/app/.heroku/python/lib/python3.9/site-packages/django/core/management/__init__.py", line 413, in execute

    self.fetch_command(subcommand).run_from_argv(self.argv)

  File "/app/.heroku/python/lib/python3.9/site-packages/django/core/management/base.py", line 354, in run_from_argv

    self.execute(*args, **cmd_options)

  File "/app/.heroku/python/lib/python3.9/site-packages/django/core/management/base.py", line 398, in execute

    output = self.handle(*args, **options)

  File "/app/pontoon/base/management/commands/heroku_deploy_setup.py", line 23, in handle

    Project.objects.filter(slug="pontoon-intro").update(

  File "/app/.heroku/python/lib/python3.9/site-packages/django/db/models/query.py", line 779, in update

    query.add_update_values(kwargs)

  File "/app/.heroku/python/lib/python3.9/site-packages/django/db/models/sql/subqueries.py", line 85, in add_update_values

    field = self.get_meta().get_field(name)

  File "/app/.heroku/python/lib/python3.9/site-packages/django/db/models/options.py", line 610, in get_field

    raise FieldDoesNotExist("%s has no field named '%s'" % (self.object_name, field_name))

django.core.exceptions.FieldDoesNotExist: Project has no field named 'url'

[0m

```

**Comment by mathjazz** [comment]
Thanks foe clarifying!

You mention that you had to install mysqlclient, but mysqlclient is not a requirement for Pontoon. Since we deploy pontoon.mozilla.org to Heroku regularly without problems, I suspect this issue could be caused by bad configuration.

Did you perhaps configure MySQL as opposed to PostgreSQL?

**Comment by surapunoyousei** [comment]
> Thanks foe clarifying!
> 
> You mention that you had to install mysqlclient, but mysqlclient is not a requirement for Pontoon. Since we deploy pontoon.mozilla.org to Heroku regularly without problems, I suspect this issue could be caused by bad configuration.
> 
> Did you perhaps configure MySQL as opposed to PostgreSQL?

I will review the configuration and rebuild.

BTW, is deploying from a button a bug?

**Comment by mathjazz** [comment]
Yeah, we track it in #2460.

**Comment by surapunoyousei** [comment]
```
remote: Collecting static files...
remote: Traceback (most recent call last):
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/db/backends/mysql/base.py", line 15, in <module>
remote:     import MySQLdb as Database
remote: ModuleNotFoundError: No module named 'MySQLdb'
remote: 
remote: The above exception was the direct cause of the following exception:
remote: 
remote: Traceback (most recent call last):
remote:   File "/tmp/build_1f0ac2bf/./manage.py", line 16, in <module>
remote:     execute_from_command_line(sys.argv)
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
remote:     utility.execute()
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/core/management/__init__.py", line 395, in execute
remote:     django.setup()
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/__init__.py", line 24, in setup
remote:     apps.populate(settings.INSTALLED_APPS)
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/apps/registry.py", line 114, in populate
remote:     app_config.import_models()
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/apps/config.py", line 301, in import_models
remote:     self.models_module = import_module(models_module_name)
remote:   File "/app/.heroku/python/lib/python3.9/importlib/__init__.py", line 127, in import_module
remote:     return _bootstrap._gcd_import(name[level:], package, level)
remote:   File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
remote:   File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
remote:   File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
remote:   File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
remote:   File "<frozen importlib._bootstrap_external>", line 850, in exec_module
remote:   File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
remote:   File "/tmp/build_1f0ac2bf/pontoon/actionlog/models.py", line 5, in <module>
remote:     class ActionLog(models.Model):
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/db/models/base.py", line 122, in __new__
remote:     new_class.add_to_class('_meta', Options(meta, app_label))
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/db/models/base.py", line 326, in add_to_class
remote:     value.contribute_to_class(cls, name)
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/db/models/options.py", line 207, in contribute_to_class
remote:     self.db_table = truncate_name(self.db_table, connection.ops.max_name_length())
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/utils/connection.py", line 15, in __getattr__
remote:     return getattr(self._connections[self._alias], item)
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/utils/connection.py", line 62, in __getitem__
remote:     conn = self.create_connection(alias)
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/db/utils.py", line 204, in create_connection
remote:     backend = load_backend(db['ENGINE'])
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/db/utils.py", line 111, in load_backend
remote:     return import_module('%s.base' % backend_name)
remote:   File "/app/.heroku/python/lib/python3.9/importlib/__init__.py", line 127, in import_module
remote:     return _bootstrap._gcd_import(name[level:], package, level)
remote:   File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
remote:   File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
remote:   File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
remote:   File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
remote:   File "<frozen importlib._bootstrap_external>", line 850, in exec_module
remote:   File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/db/backends/mysql/base.py", line 17, in <module>
remote:     raise ImproperlyConfigured(
remote: django.core.exceptions.ImproperlyConfigured: Error loading MySQLdb module.
remote: Did you install mysqlclient?
remote: Traceback (most recent call last):
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/db/backends/mysql/base.py", line 15, in <module>
remote:     import MySQLdb as Database
remote: ModuleNotFoundError: No module named 'MySQLdb'
remote: 
remote: The above exception was the direct cause of the following exception:
remote: 
remote: Traceback (most recent call last):
remote:   File "/tmp/build_1f0ac2bf/./manage.py", line 16, in <module>
remote:     execute_from_command_line(sys.argv)
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/core/management/__init__.py", line 419, in execute_from_command_line
remote:     utility.execute()
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/core/management/__init__.py", line 395, in execute
remote:     django.setup()
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/__init__.py", line 24, in setup
remote:     apps.populate(settings.INSTALLED_APPS)
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/apps/registry.py", line 114, in populate
remote:     app_config.import_models()
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/apps/config.py", line 301, in import_models
remote:     self.models_module = import_module(models_module_name)
remote:   File "/app/.heroku/python/lib/python3.9/importlib/__init__.py", line 127, in import_module
remote:     return _bootstrap._gcd_import(name[level:], package, level)
remote:   File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
remote:   File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
remote:   File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
remote:   File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
remote:   File "<frozen importlib._bootstrap_external>", line 850, in exec_module
remote:   File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
remote:   File "/tmp/build_1f0ac2bf/pontoon/actionlog/models.py", line 5, in <module>
remote:     class ActionLog(models.Model):
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/db/models/base.py", line 122, in __new__
remote:     new_class.add_to_class('_meta', Options(meta, app_label))
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/db/models/base.py", line 326, in add_to_class
remote:     value.contribute_to_class(cls, name)
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/db/models/options.py", line 207, in contribute_to_class
remote:     self.db_table = truncate_name(self.db_table, connection.ops.max_name_length())
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/utils/connection.py", line 15, in __getattr__
remote:     return getattr(self._connections[self._alias], item)
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/utils/connection.py", line 62, in __getitem__
remote:     conn = self.create_connection(alias)
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/db/utils.py", line 204, in create_connection
remote:     backend = load_backend(db['ENGINE'])
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/db/utils.py", line 111, in load_backend
remote:     return import_module('%s.base' % backend_name)
remote:   File "/app/.heroku/python/lib/python3.9/importlib/__init__.py", line 127, in import_module
remote:     return _bootstrap._gcd_import(name[level:], package, level)
remote:   File "<frozen importlib._bootstrap>", line 1030, in _gcd_import
remote:   File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
remote:   File "<frozen importlib._bootstrap>", line 986, in _find_and_load_unlocked
remote:   File "<frozen importlib._bootstrap>", line 680, in _load_unlocked
remote:   File "<frozen importlib._bootstrap_external>", line 850, in exec_module
remote:   File "<frozen importlib._bootstrap>", line 228, in _call_with_frames_removed
remote:   File "/app/.heroku/python/lib/python3.9/site-packages/django/db/backends/mysql/base.py", line 17, in <module>
remote:     raise ImproperlyConfigured(
remote: django.core.exceptions.ImproperlyConfigured: Error loading MySQLdb module.
remote: Did you install mysqlclient?
remote: Current commit being deployed: 
remote: 0cd238e65bb4aa2629e1229d828b9886e7a56f66

```


Sorry, but the problem still occurs.

I have re-forked, no normal commits, added only the required values that are shown when clicking the Deployment button to heroku and pushed, but the error still occurs.

Do you have any idea what is going on?

**Comment by creeper-0910** [comment]
https://github.com/creeper-0910/pontoon/commit/f8206ddf1563a6cf5e7af961efe38c1563086b2d
I removed the part where the error was occurring and it started up normally.
What is this code used for...?

**Comment by marcellov7** [comment]
Confirmed, it does not work. Same "url" error.

**Comment by mathjazz** [comment]
Adding for reference that we have fixed #2460 and Deploy to Heroku button now works.

**Comment by radziejewicz** [comment]
In docker compose same.
